lib/module.js

```diff
@@ -44,7 +47,7 @@ function axiosModule(_moduleOptions) {
     proxyHeadersIgnore: ['accept', 'host', 'cf-ray', 'cf-connecting-ip', 'content-length'],
     proxy: false,
     retry: false,
-    https: false,
+    https: https,
```

Comment:
```suggestion
    https,
```

Using the shorthand here would cleaner IMO :relaxed:

Reply:
I did consider this, looks good.

Reply:
refactor(https): use shorthand in options